articles/install-guide/csinstall.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Develop with Q# + C#
+author: natke
+ms.author: nakersha
+ms.date: 9/30/2019
+ms.topic: article
+ms.custom: how-to
+uid: microsoft.quantum.install.cs
+---
+# Develop with Q# + C#
+
+This is the installation guide for the QDK environment to program hybrid Q# and C# programs.
```

Comment:
Suggest more active language:

"Install the QDK to develop C# host programs to call Q# operations" or similar